### PR TITLE
OADP-193: Fix restic enablement with custom BSL secret

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: pkg/credentials/credentials.go [hunk 1]
```diff
@@ -156,6 +156,16 @@ func AppendCloudProviderVolumes(dpa *oadpv1alpha1.DataProtectionApplication, ds
 				continue
 			}
 
+			pluginNeedsCheck, foundInBSLorVSL := providerNeedsDefaultCreds[string(plugin)]
```
Comment (shawn-hurley) [inline pkg/credentials/credentials.go:159]: Do we have to make sure to strip out velero.io here?

**Review by shawn-hurley — APPROVED** [review]
LGTM just a little question

File: pkg/credentials/credentials.go (hunk 1 above)
Comment (dymurray) [reply pkg/credentials/credentials.go:159]: We do not, this gets stripped in `noDefaultCredentials` func

**Comment by dymurray** [comment]
Now also included additional fix for OADP 150

**Review by shawn-hurley — APPROVED** [review]

**Pull request merged**